pythonFiles/tests/debug_adapter/test_ptvsd_folder_name_functional.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+import subprocess
+import sys
+
+from os import path
+from packaging.requirements import Requirement
+
+
+ROOT = path.dirname(path.dirname(path.dirname(path.dirname(path.abspath(__file__)))))
+PYTHONFILES_ROOT = path.join(ROOT, "pythonFiles")
+PYTHONFILES = path.join(PYTHONFILES_ROOT, "lib", "python")
+REQUIREMENTS = path.join(ROOT, "requirements.txt")
+ARGV = ["python", path.join(PYTHONFILES_ROOT, "ptvsd_folder_name.py")]
+
+
+def isPython37():
+    return sys.version_info.major == 3 and sys.version_info.minor == 7
+
+
+@pytest.mark.skipif(not isPython37(), reason="PTVSD wheels shipped for Python 3.7 only")
+@pytest.mark.functional
+class TestPtvsdFolderNameFunctional(object):
+    """Functional tests for the script retrieving the PTVSD folder name for the PTVSD wheels experiment."""
+
+    @classmethod
+    def setup_class(cls):
+        cls.version = cls.ptvsd_version()
+
+    def ptvsd_version():
+        with open(REQUIREMENTS, "r", encoding="utf-8") as reqsfile:
+            for line in reqsfile:
+                pkgreq = Requirement(line)
+                if pkgreq.name == "ptvsd":
+                    specs = pkgreq.specifier
+                    return next(iter(specs)).version
+
+    def ptvsd_paths(self, platforms=[]):
```

Comment:
Consider using `*args`:

```suggestion
    def ptvsd_paths(self, *platforms):
```